microsoft/testsuites/core/timesync.py

```diff
@@ -12,9 +12,10 @@
     TestSuiteMetadata,
     create_timer,
 )
-from lisa.operating_system import Redhat
+from lisa.operating_system import Architecture, Redhat
 from lisa.tools import Cat, Chrony, Dmesg, Hwclock, Lscpu, Ntp, Ntpstat, Service
 from lisa.tools.lscpu import CpuType
+from lisa.util import UnsupportedArchException
```

Comment:
Add it to lisa root package like other exceptions.

Reply:
Done